### ISSUE TITLE
download speed 

### ISSUE THREAD

**Issue by FLASH-07** [jmir1/aniyomi#270]
### Describe your suggested feature

"it's download speed is too slow it's just around 500 kbps so we can't download episodes fast enough. so please increase speed limit"

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.12.3.5](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by joelchrono12** [comment]
I dont really know, but this seems kinda intended.

If Aniyomi takes too much bandwidth it might get blocked by the servers.

It might be an actual bug tho, so idk

**Comment by jmir1** [comment]
the site itself is limiting the download speed
